spec/fixtures/valid_excluded_fields.yml

```diff
@@ -0,0 +1,6 @@
+---
+Account:
+  - Foo__c
+
+Invoicie:
```

Comment:
Typo

Reply:
Yep. Thank you for pointing it out. The test is only checking if the parsed YAML `is_a?(Hash)` so that's why it did not pick it up.